smoke-tests/apps/SnippetInjection/src/smokeTest/resources/applicationinsights.json

```diff
@@ -0,0 +1,16 @@
+{
+  "connectionString": "",
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "javaScriptSnippet":
```

Comment:
should this be "browserSdkLoader", saw your doc update?

Reply:
haha, yes, just updated it.